easybuild/easyconfigs/n/NAGlib/NAGlib-30.D-intel-compilers-2021.4.0.eb

```diff
@@ -0,0 +1,37 @@
+# NAG Library, Mark 30 NLL6I30DBL
+# After installation, you must obtain a license from NAG and put the license file in the
+# %(installdir)s/license folder as license.dat file.
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'Binary'
+
+name = 'NAGlib'
+version = '30.D'
+
+homepage = 'http://www.nag.co.uk'
+description = """The worlds largest collection of robust, documented, tested and maintained numerical algorithms.
+This is the library from the NAG collection of available software."""
+
+toolchain = {'name': 'intel-compilers', 'version': '2021.4.0'}
+
+source_urls = ['https://support.nag.com/downloads/impl']
+sources = ['nll6i%(version_major)s%(version_minor)sBL.tgz']
+checksums = ['6cc8c603097db7e9530487b8a955413f0a73adffd62ac83eddfafb6d6e2f3679']
+
+extract_sources = True
+
+install_cmd = "./install.sh -accept -installdir=%(installdir)s"
+
+postinstallcmds = ['cd %(installdir)s ; mv nll6i%(version_major)s*/* . && rmdir nll6i%(version_major)s* ']
```

Comment:
There is an extra space at the end here, remove it to make the two versions similar.